test/test.c

```diff
@@ -0,0 +1,8 @@
+float apple = 3.0f;
+float banana = 65.32f;
+
+int something(void) {
+    return (int)(apple * banana) % 11;
+}
+
+something();
```

Comment:
mistake?

Reply:
no, this file exists for the sake of testing. or do you just mean the function call at the end?

Reply:
yeah, the legacy-style function declaration at the end